pkg/api/types.go

```diff
@@ -1817,6 +1817,7 @@ const (
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
 	// Number of Pods that may be running on this Node: see ResourcePods
```

Comment:
It's a placeholder for the defunct `maxpods`. It misled me into thinking I had already added a description for ResourceNvidiaGPU, so I just moved it to the end.